src/Models/RaspberryPiDevice.ts

```diff
@@ -42,6 +42,32 @@ export class RaspberryPiDevice extends ContainerDeviceBase {
         DeviceType.Raspberry_Pi, templateFilesInfo);
   }
 
+  private async getBinaryFileName(): Promise<string|undefined> {
+    // Parse binary name from CMakeLists.txt file
+    const cmakeFilePath = path.join(this.projectFolder, 'CMakeLists.txt');
+    if (!await FileUtility.fileExists(ScaffoldType.Workspace, cmakeFilePath)) {
+      return;
+    }
+    const getBinaryFileNameCmd = `cat ${
+        cmakeFilePath} | grep 'set(binary_name' | cut -d ' ' -f2 | sed -e 's/^"//' -e 's/"$//' | tr -d '\n'`;
+
+    const binaryName = await execute(getBinaryFileNameCmd);
+    return binaryName;
+  }
+
+  private async EnableBinaryExecutability(

```

Comment:
Naming Consistency: first letter should be lower case

Reply:
Fixed.